warehouse/seeds/gtfs_rt_validation_code_descriptions.csv

```diff
@@ -0,0 +1,58 @@
+Code,Description,Is_Critical
```

Comment:
is it possible to comment where this came from, especially version information?

Reply:
yep will document/test seeds